themes/app/templates/App/PageType/Layout/ContentPage.ss

```diff
@@ -0,0 +1,12 @@
+<% include Breadcrumb %>
```

Comment:
This top section of thhe content page can be turned into a generic include called `PageHeader.ss` 

Think other page types have the potential of reusing this page header so it would be good to make it reusable by using an include.

Reply:
You can simply call `$Breadcrumbs` here and it should render your template :)